pybossa/jobs.py

```diff
@@ -727,8 +728,11 @@ def send_email_notifications():
             body = u'Hello,\n\nThere have been new tasks uploaded to the previously finished project, {0}. ' \
                    u'\nLog on to {1} to complete any available tasks.' \
                 .format(project.name, current_app.config.get('BRAND'))
-            mail_dict = dict(recipients=recipients, subject=subject, body=body)
-            send_mail(mail_dict)
+            recipients_chunk = [recipients[x * MAX_RECIPIENTS : (x+1) * MAX_RECIPIENTS]
+                                for x in range(0, int(math.ceil(len(recipients)*1.0 / MAX_RECIPIENTS)))]
```

Comment:
Seems, this can be simplified as
```python
recipients_chunks =[ recipients[i:i + MAX_RECIPIENTS] for i in range(0, len(recipients), MAX_RECIPIENTS) ]
```